springfox-core/src/main/java/springfox/documentation/builders/OperationBuilder.java

```diff
@@ -232,9 +233,20 @@ public OperationBuilder tags(Set<String> tags) {
     return this;
   }
 
+  /**
+   * Sets the nickname for the operation
+   *
+   * @param nickname - a string for the nickname
+   * @return this
+   */
+  public OperationBuilder nickname(String nickname) {
+      this.nickname = defaultIfAbsent(nickname, this.nickname);
+      return this;
+  }
+
   public Operation build() {
     return new Operation(method, summary, notes, responseModel, uniqueId, position, tags, produces,
-        consumes, protocol, securityReferences, parameters, responseMessages, deprecated, isHidden);
+        consumes, protocol, securityReferences, parameters, responseMessages, deprecated, isHidden, nickname);
```

Comment:
There is already a uniqueId in the operation model. I dont think we want to introduce swagger specific vocabulary into our internal models! If we change this I think this pr can be :merged: :+1: